src/main/java/com/example/petbuddybackend/controller/OfferController.java

```diff
@@ -45,6 +45,17 @@ public OfferDTO addOrEditOffer(@RequestBody @Valid ModifyOfferDTO offer,
         return offerService.addOrEditOffer(offer, principal.getName());
     }
 
+    @DeleteMapping("/{offerId}")
+    @PreAuthorize("isAuthenticated()")
+    public OfferDTO deleteOffer(@PathVariable Long offerId,
+                                Principal principal,
+
+                                @RoleParameter
+                                @AcceptRole(acceptRole = Role.CARETAKER)
```

Comment:
Może dodajmy to `RoleParameter` do `AcceptRole`. Jest to u mnie na PR ale też nie wiadomo kiedy to zmerguje. A tak to na nowych annotacjach już nie będziemy tego dodawać

Reply:
I jak coś ze starych roli nie trzeba usuwać bo to już jest na tym PR. Byleby na nowych nie dodawać sobie roboty

Reply:
w sumie tak